docs/source/whatsnew/1.3.0.txt

```diff
@@ -0,0 +1,90 @@
+Development
+-----------
+
+:Release: 1.3.0
+:Date: TBD
+
+Highlights
+~~~~~~~~~~
+
+Support for Newer Numpy/Pandas Versions
+```````````````````````````````````````
+
+Zipline has historically been very conservative when updating versions of
+numpy, pandas, and other "PyData" ecosystem packages. This conservatism is
+primarily due to the fact that Zipline is used as the backtesting engine for
+[Quantopian](https://www.quantopian.com/), which means that updating package
+versions risks breaking a large installed codebase. Of course, many Zipline
+users don't have the backwards compatibility requirements that Quantopian has,
+and they'd like to be able to use the latest and greatest package versions.
+
+As part of this release, we're now building and testing Zipline with two
+package configurations:
+
+- "Stable", using numpy version 1.11 and pandas version 0.18.1.
+- "Latest", using numpy version 1.14 and pandas version 0.22.0.
+
+Other combinations of numpy and pandas **may** work, but these package sets
+will be tested during our normal build/release cycle.
+
+Moving forward, our goal to continue to maintain support for two sets of
+packages at any given time. The "stable" package set will change relatively
+infrequently, and will contain the versions of numpy and pandas supported on
+Quantopian. The "latset" package set will change regularly, and will contain
+recently-released versions of numpy and pandas.
+
+Our hope with these changes is to strike a balance between stability and
+novelty without taking on too great a maintenance burden by supporting every
+possible combination of packages.
+
+Standalone `trading_calendars` Module
```

Comment:
I think the link here is broken because on line 58 we have:

```
`trading-calendars`_ package
```

and the link is:

```
 _`trading_calendars`
```

I'm thinking we want all mentions to link to the same place.